src/view/add-new-trip-button-view.js

```diff
@@ -0,0 +1,23 @@
+import {createElement} from '../render.js';
+
+const addNewButtonTemplate = () =>
+  `<button class="trip-main__event-add-btn  btn  btn--big  btn--yellow" type="button">New event</button>
+`;
+
+export default class AddNewButtonView {
```

Comment:
[Критерий Б27](https://up.htmlacademy.ru/ecmascript-individual/1/criteries#b27):
```suggestion
export default class AddNewTripButtonView {
```